### PR TITLE
Update: Function to populate cypress test spec if no argument value passed [3595]

### PR DESCRIPTION
**Pull request by cahirodoherty-learningpool**
[//]: # (Please title your PR according to eslint commit conventions)
[//]: # (See https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-eslint#eslint-convention for details)

[//]: # (Link the PR to the original issue)
Addresses #3595 

[//]: # (Delete Fix, Update, New and/or Breaking sections as appropriate)
### Update
* Function to populate cypress test spec if no argument value passed

### CODE REVIEW

**Review by ethan-lp — APPROVED** [review]

**Review by lemmyadams — APPROVED** [review]

**Comment by oliverfoster** [comment]
Are you doing the jest equivalent in another pr?

File: test.js [hunk 1]
```diff
@@ -80,12 +81,24 @@ async function waitForGruntServer() {
...
-  if (argumentValues.testfiles) {
-    return asyncSpawn('node', './node_modules/cypress/bin/cypress', 'run', '--spec', `${argumentValues.testfiles}`, '--config', `{"fixturesFolder": "${argumentValues.outputdir}"}`);
-  }
+async function populateTestFiles() {
+  // accept the user-specified file(s)
+  if (argumentValues.testfiles) return;
 
-  return asyncSpawn('node', './node_modules/cypress/bin/cypress', 'run', '--config', `{"fixturesFolder": "${argumentValues.outputdir}"}`);
+  // otherwise, only include test files for plugins present in the course config
+  const config = JSON.parse(fs.readFileSync('./build/course/config.json'));
```
Comment (oliverfoster) [inline test.js:89]:
```js
  const config = JSON.parse(fs.readFileSync('./build/course/config.json'));
```
Hard coded `./build/` ignores that outputdir is definable.

Comment (cahirodoherty-learningpool) [reply test.js:89]: Good catch, thanks Oliver

**Comment by cahirodoherty-learningpool** [comment]
> Are you doing the jest equivalent in another pr?

Hi @oliverfoster ,
Sorry, I somehow overlooked this comment but I have actioned now 👍

**Review by oliverfoster — APPROVED** [review]

**Pull request merged**